portfolio/src/main/webapp/index.html

```diff
@@ -6,22 +6,19 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+<body>
     <div id="content">
       <h1>Megan Shi</h1>
       <img src="images/me.JPG" height=250/>
```

Comment:
img tag ends with `>` only please remove the `/`
also please add an alt for all images

https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback